pages/deploy/deploy_aws.md

```diff
@@ -235,6 +237,8 @@ and find 2 examples of cdk.json files.
 | source                                        | Optional              | The version control source for the repository. It can take 2 values 'codecommit' or 'codestar_connection'. (default: 'codecommit')                                                                                                                                                                                                                                                                                   |
 | repo_string                                   | Optional              | The repository path as string. Required if source='codestar_connection' (default: 'awslabs/aws-dataall')                                                                                                                                                                                                                                                                                                             |
 | repo_connection_arn                           | Optional              | The arn of the CodeStar connection connecting with the source repository. Required if source='codestar_connection'(default: None)                                                                                                                                                                                                                                                                                    |
+| log_retention_duration                        | Optional              | The CloudWatch log retention days - as a string enum value one of AWS CDK RetentionDays members (default: `TWO_YEARS`) |
```

Comment:
it would be good to mention for which log groups - all data.all deployed log groups, environment accounts, infra accounts?